jujugui/static/gui/src/app/init/changes-utils.js

```diff
@@ -213,19 +231,24 @@ const changesUtils = {
           changeItem.icon = ghostService.get('icon');
           changeItem.description = ' ' + ghostService.get('name') +
               ' will be added to the model.';
+          changeItem.command = `juju deploy ${ghostService.get('charm')}`;
```

Comment:
Should we add the app name here? Or maybe only in the case it's different from the charm name?

Reply:
Good point, done.